pom.xml

```diff
@@ -205,4 +211,20 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <configuration>
+                        <systemPropertyVariables>
+                            <java.util.logging.config.file>src/test/resources/logging.properties</java.util.logging.config.file>
```

Comment:
You do not need to do this. Use https://jenkinsci.github.io/maven-hpi-plugin/run-mojo.html#loggers instead.

Reply:
when running test? 
As some tests were flaky I needed some way to have logs when running tests in https://ci.jenkins.io/

Reply:
Oh sorry this is about test cases; the above is for `mvn hpi:run`. Use https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/LoggerRule.html

Reply:
ok I changed that too.
But it means this need to add some coding in all test classes which is a bit less convenient than changing a single file....